README.md

```diff
@@ -56,7 +56,26 @@ The whole purpose of this stage of the process is to look for meaningful primary
 The relationships between artefacts will be defined in detail by the indicators that the Querying and ML components will detect/calculate. The first step of relationships discovery is finding good indicators of relationships between the different artefacts. These indicators are extracted by processing the analytic results from Holmes Totem (Dynamic). The components responsible for performing this analytic processes are the Query and ML components.
 
 ###### Final Relationships
+The final relationships are relationships among Malware, Domain and IP. These relationships will look as follows: 
```

Comment:
You are using relationships to explain relationships. I would recommend phrasing this differently. Maybe state what the relationships mean. i.e. The final relationships define how objects in the system are associated with each other. These are created by analyzing the primary relationships and determining if and how strongly objects are related. Currently, we focus on samples, domains, and IPs and seek to identify the following: